Terminal.Gui/ConsoleDrivers/EscSeqUtils/EscSeqUtils.cs

```diff
@@ -461,7 +461,7 @@ public static ConsoleModifiers GetConsoleModifiers (string value)
     /// <returns>The c1Control returned by <see cref="GetC1ControlChar(char)"/>, code, values and terminating.</returns>
     public static (string c1Control, string code, string [] values, string terminating) GetEscapeResult (char [] kChar)
     {
-        if ((kChar == null) || (kChar.Length == 0))
+        if (kChar == null || kChar.Length == 0)
```

Comment:
Another that might be able to be handled by a rule:

`if (kChar is not { Length: > 0})`

or using spans:

`if (kChar is not [_,..])`